### ISSUE TITLE
Applied Energistics freezes with more than one drive.

### ISSUE THREAD

**Issue by david7867z** [DarkPacks/SkyFactory-4#1217]
## Bug Report

The ME systems terminal freezes when more than one drive is connected.
I am showing this in this video: https://youtu.be/Byno9YOxqXY

## Steps to Reproduce (for bugs)

1. Set up an ME System
2. Connect Drive
3. Acess System normally.
4. Connect second drive
5. cant acess via terminal, only wirelessly

## Logs
 The game isnt crashing!

* Client/Server Log:
* Crash Log:

## World Information

* Preset: Default Skyblock

* Prestige: No
 1.12.2
* Modpack Version world created in: 1.12.2

* Additional Content Installed: None

## Client Information

* Modpack Version: 1.12.2
* Java Version: 1.8
* Launcher Used: Twitch
* Memory Allocated: 12GB
* Server/LAN/Single Player: LAN
* Optifine Installed: True
* Shaders Enabled: True

**Comment by sk2048** [comment]
My guess is the network has too small of a power buffer to allow you to withdraw items through the terminal. Add more (Dense) Energy Cells to your AE network.

**Comment by david7867z** [comment]
But wouldnt it affect the wireless terminal too? I will try it out later. Thanks.

**Comment by sk2048** [comment]
No, wireless terminal pays the energy cost from the item’s energy I think.
